### PR TITLE
Restore Team page 

### PR DESCRIPTION
**Pull request by Jkovarik**

### CODE REVIEW

File: docs/team.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+id: team
+title: Cumulus Tasks
```
Comment (laurenfrederick) [inline docs/team.md:3]:
```suggestion
title: Cumulus Team
```

Comment (Jkovarik) [reply docs/team.md]: 🤦  thanks

**Comment by Jkovarik** [comment]
Updated to restore team page from https://github.com/nasa/cumulus/pull/1738

File: docs/team.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+id: team
+title: Cumulus Team
+hide_title: true
+---
+
+### Cumulus Core Team
+
+- Chuck Daniels ([@chuckwondo](https://github.com/chuckwondo))
```
Comment (laurenfrederick) [inline docs/team.md:9]:
```suggestion
- Brian Tennity ([@bt6453](https://github.com/bt8453))
- Chuck Daniels ([@chuckwondo](https://github.com/chuckwondo))
- Daniel Peters ([@dopeters](https://github.com/dopeters))
```

**Pull request merged**